src/synthesizer_grids/incident/sps/install_maraston24.py

```
@@ -136,7 +130,31 @@ def make_grid(model, rotation, model_type, imf, input_dir, grid_dir):
     for imf in imfs:
         for model_type in model_types:
             for rotation in rotations:
-                print(imf, model_type, rotation)
+                
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)